src/state/StateInit.cpp

```diff
@@ -116,6 +116,7 @@ namespace polyfem
 			{"t0", 0},
 			{"tend", 1},
 			{"time_steps", 10},
+			{"skip_frame", 1},
```

Comment:
what it this

Reply:
Ok got it, can you add to the other saves?

Reply:
Sorry I don't understand

Reply:
add this `if (args["save_time_sequence"] && !(t % (int)args["skip_frame"]))` to the other time dependent solves :)